From: Hannes Reinecke <hare@suse.de>
Date: Fri, 17 Aug 2018 14:02:11 +0200
Subject: [PATCH] nvme-multipath: disable native NVMe multipath per default
References: bsc#1105151
Patch-Mainline: never, SLE12 SP4 specific

For backwards compability we cannot enable native NVMe multipathing,
but rather have to require the admin to change the module parameter.

Signed-off-by: Hannes Reinecke <hare@suse.com>
---
 drivers/nvme/host/multipath.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/nvme/host/multipath.c b/drivers/nvme/host/multipath.c
index c1f41aa879c1..2b1aa0119359 100644
--- a/drivers/nvme/host/multipath.c
+++ b/drivers/nvme/host/multipath.c
@@ -15,10 +15,10 @@
 #include <trace/events/block.h>
 #include "nvme.h"
 
-static bool multipath = true;
+static bool multipath = false;
 module_param(multipath, bool, 0444);
 MODULE_PARM_DESC(multipath,
-	"turn on native support for multiple controllers per subsystem");
+	"native support for multiple controllers per subsystem (default: off)");
 
 inline bool nvme_ctrl_use_ana(struct nvme_ctrl *ctrl)
 {
-- 
2.12.3

